docs/standard/components.md

```diff
@@ -78,6 +78,26 @@ You have access to an extensive set of tools and infrastructure components that
 - [NuGet](/nuget/), Microsoft's package manager for .NET
 - Open-source build orchestration tools, such as [CAKE](https://cakebuild.net/) and [FAKE](https://fake.build/)
 
+## Applicable standards
+
+The C# Language and the Common Language Infrastructure (CLI) specifications are standardized through [Ecma International®](http://www.ecma-international.org/). The first editions of these standards were published by Ecma in December 2001.
+
+Subsequent revisions to the standards have been developed by the TC49-TG2 (C#) and TC49-TG3 (CLI) task groups within the Programming Languages Technical Committee ([TC49](http://www.ecma-international.org/memento/tc49.htm)), and adopted by the Ecma General Assembly and subsequently by ISO/IEC JTC 1 via the ISO Fast-Track process.
+
+### Latest Standards
+
+The following official Ecma documents are available for [C#](http://www.ecma-international.org/publications/standards/Ecma-334.htm) and the [CLI](http://www.ecma-international.org/publications/standards/Ecma-335.htm) ([TR-84](http://www.ecma-international.org/publications/techreports/E-TR-084.htm)). These links are direct from Ecma International®:
+
+- **The C# Language Standard (version 5.0)**: [ECMA-334.pdf](http://www.ecma-international.org/publications/files/ECMA-ST/ECMA-334.pdf)
+- **The Common Language Infrastructure**: This is avaialble in [pdf](http://www.ecma-international.org/publications/files/ECMA-ST/ECMA-335.pdf) form and [zip](http://www.ecma-international.org/publications/files/ECMA-ST/ECMA-335.zip) form.
```

Comment:
```suggestion
- **The Common Language Infrastructure**: This is avaialble in [pdf](https://www.ecma-international.org/publications/files/ECMA-ST/ECMA-335.pdf) form and [zip](https://www.ecma-international.org/publications/files/ECMA-ST/ECMA-335.zip) form.
```

Reply:
I've added a typo fix in your suggestion as well @Youssef1313.